### PR TITLE
Fix | Addressing documentation issues along with spelling errors

### PR DESCRIPTION
**Pull request by JRahnama**
This PR tries to address some of the spelling errors in this repository and also addresses some of documentation issues with paramref.

Note: I used [CSpell](https://cspell.org/) to check spellings, there are other tools that could be used. To add more, a GitHub action could be set for spelling errors on each PR. Just for future references.

### CODE REVIEW

File: tools/targets/ClientInfo.targets
Comment (ErikEJ) [inline tools/targets/ClientInfo.targets:1]: Was this added by mistake?

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SSRP.cs [hunk 1]
```diff
@@ -383,11 +383,11 @@ private static SsrpResult SendUDPRequest(IPEndPoint endPoint, byte[] requestPack
         }
 
         /// <summary>
-        /// Sends request to server, and recieves response from server (SQLBrowser) on port 1434 by UDP
+        /// Sends request to server, and receives response from server (SQLBrowser) on port 1434 by UDP
         /// Request (https://docs.microsoft.com/en-us/openspecs/windows_protocols/mc-sqlr/a3035afa-c268-4699-b8fd-4f351e5c8e9e)
         /// Response (https://docs.microsoft.com/en-us/openspecs/windows_protocols/mc-sqlr/2e1560c9-5097-4023-9f5e-72b9ff1ec3b1) 
         /// </summary>
-        /// <returns>string constaning list of SVR_RESP(just RESP_DATA)</returns>
+        /// <returns>string contaning list of SVR_RESP(just RESP_DATA)</returns>
```
Comment (mdaigle) [inline src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SSRP.cs:390]:
```suggestion
        /// <returns>string containing list of SVR_RESP(just RESP_DATA)</returns>
```

**Review by mdaigle — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1254,7 +1254,7 @@ This update brings the below changes over the previous release:
 - Improved performance of Managed SNI by enhancing utilization of resources [#173](https://github.com/dotnet/SqlClient/pull/173) - Ported [dotnet/corefx#35363](https://github.com/dotnet/corefx/pull/35363) and [dotnet/corefx#40732](https://github.com/dotnet/corefx/pull/40732)
 - Improved performance of Managed SNI RPC Parameter Usage [#209](https://github.com/dotnet/SqlClient/pull/209) - Ported [dotnet/corefx#34049](https://github.com/dotnet/corefx/pull/34049)
 - Changed enclave key map to be lazy initialized [#372](https://github.com/dotnet/SqlClient/pull/372)
-- Changed `Recieve()` and `ReceiveAsync()` implementation to receive null packets on failure [#350](https://github.com/dotnet/SqlClient/pull/350)
+- Changed `receive()` and `ReceiveAsync()` implementation to receive null packets on failure [#350](https://github.com/dotnet/SqlClient/pull/350)
```
Comment (JRahnama) [inline CHANGELOG.md:1257]:
```suggestion
- Changed `Receive()` and `ReceiveAsync()` implementation to receive null packets on failure [#350](https://github.com/dotnet/SqlClient/pull/350)
```

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SslOverTdsStream.cs [hunk 3]
```diff
@@ -99,7 +99,7 @@ public override long Position
         public override bool CanWrite => _stream.CanWrite;
 
         /// <summary>
-        /// Check if stream can be seeked
+        /// Check if stream can be sought
```
Comment (David-Engel) [inline src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SslOverTdsStream.cs:102]:
```suggestion
        /// Check if stream can seek
```

**Comment by JRahnama** [comment]
I wanted to mention that there are some spelling errors in the function names, as well as likely more errors in the documentation that I didn’t have time to review. To identify spelling mistakes in function names, you can run `cspell src/**/*.cs` or `cspell "src/**/*.xml` to catch the rest. You can also save the results in a .txt file by using the appropriate command with below command:

`cspell --words-only --unique "**/*.xml" | sort --ignore-case >> results.txt`

**Review by mdaigle — APPROVED** [review]

**Review by David-Engel — APPROVED** [review]

**Pull request merged**